terminology/ansi.py

```diff
@@ -211,7 +219,7 @@ def _remove_underline(text) -> StyledText:
 STYLE_RESET = "\033[0m"
 FOREGROUND_COLORS_REGEX = "\033\\[3[0-9]m"
 BACKGROUND_COLORS_REGEX = "\033\\[4[0-9]m"
-BOLD_REGEX = "\033\\[1m"
+BOLD_OR_DIM_REGEX = "\033\\[[1-2]m"
```

Comment:
So, I tested this out, and there is a subtle difference between dimmed/bold and dimmed and bold:

![Screen Shot 2021-02-23 at 5 27 54 PM](https://user-images.githubusercontent.com/17055383/108922346-91027d80-75fc-11eb-82dc-af94f72f37ec.png)

Reply:
I have no idea why my terminal would make this distinction lol, but might as well just allow for text to be both bold and dimmed

Reply:
Actually, I can see why you may want text to be bold and dimmed, here's an example to show the difference:

![Screen Shot 2021-02-23 at 5 31 38 PM](https://user-images.githubusercontent.com/17055383/108922668-1128e300-75fd-11eb-9fd7-0878721a381a.png)